sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonExporter.java

```diff
@@ -51,7 +53,7 @@
 public class GeoJsonExporter extends Thread{
 
     private static final String TAG = GeoJsonExporter.class.getSimpleName();
-    private static final int RENDER_JSON = 1;
+    private static final int RENDER_JSON = 2; //GEOJSON
```

Comment:
This comment isn't very clear. Does the value 2 signal to the renderer it is geojson?

Reply:
yes